pkg/util/tags.go

```diff
@@ -0,0 +1,66 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package util
+
+import (
+	"strings"
+	"time"
+
+	ackconfig "github.com/aws/aws-controllers-k8s/pkg/config"
+)
+
+// GetDefaultTags provides Default tags (key value pairs)
+func GetDefaultTags(config *ackconfig.Config) map[string]string {
+	if config == nil || config.ResourceTags == nil || len(config.ResourceTags) == 0 {
+		return nil
+	}
+	var populatedTags = make(map[string]string)
+	for _, tagKeyVal := range config.ResourceTags {
+		keyVal := strings.Split(tagKeyVal, "=")
+		if keyVal == nil && len(keyVal) != 2 {
+			continue
+		}
+		key := strings.TrimSpace(keyVal[0])
+		val := strings.TrimSpace(keyVal[1])
+		if key == "" || val == "" {
+			continue
+		}
+		populatedValue := expandTagValue(&val)
+		populatedTags[key] = *populatedValue
+	}
+	if len(populatedTags) == 0 {
+		return nil
+	}
+	return populatedTags
+}
+
+func expandTagValue(value *string) *string {
```

Comment:
This will likely need to accept the CR as a parameter (in order to expand things like KUBERNETES_NAMESPACE).

Reply:
Updated code.